drivers/sensor/bme280/bme280.c

```diff
@@ -173,6 +173,53 @@ static int bme280_sample_fetch(struct device *dev, enum sensor_channel chan)
 	if (data->chip_id == BME280_CHIP_ID) {
 		size = 8;
 	}
+
+#if defined CONFIG_BME280_FORCED_MODE_ON
+
+	/*
+	 * According to datasheet, mode bits (in force mode) must be set
+	 * before measurement starts.
+	 */
+
+	ret = bm280_reg_read(data, BME280_REG_CTRL_MEAS, buf, 1);
+	if (ret < 0) {
+		return ret;
+	}
+
+	buf[0] &= ~BME280_MODE_MASK;
+	buf[0] |= BME280_MODE_FORCED;
+
+	ret = bm280_reg_write(data, BME280_REG_CTRL_MEAS, buf[0]);
+	if (ret < 0) {
+		return ret;
+	}
```

Comment:
Replace `bm280_reg_read` and `bm280_reg_write` with `i2c_reg_update_byte`

Reply:
If I understood it right, BME280 can be wired both using SPI and I2C. Thus, `bm280_reg_write` is used.

See: https://github.com/zephyrproject-rtos/zephyr/blob/359514ad0877a91ee1d3ad4f155975085895564b/drivers/sensor/bme280/bme280.c#L74-L98

Let me know if I missed something.